Snapchat_Filters/Halloween_Pumpkin/Readme.md

```diff
@@ -0,0 +1,13 @@
+# Halloween Pumpkin Filter
+
```

Comment:
![Screenshot_20221101_034834](https://user-images.githubusercontent.com/70143208/199213410-d22e0556-309f-4be4-a83a-bde7cca314d1.png)

Reply:
@akshitagupta15june  sorry for late response.

Reply:
remove the white background from picture, take a png pumpkin image

Reply:
@akshitagupta15june  the image is png but i can't remove the white background, its my first open source contribution and working with face detection so little help would be appreciated.